.circleci/config.yml

```diff
@@ -336,6 +342,45 @@ jobs:
       - store_test_results:
           path: test-results
 
+  unittest_windows_gpu:
+    <<: *binary_common
+    executor:
+      name: windows-gpu
+    environment:
+      CUDA_VERSION: "10.1"
+    steps:
+      - checkout
+      - run:
+          name: Generate cache key
+          # This will refresh cache on Sundays, nightly build should generate new cache.
+          command: echo "$(date +"%Y-%U")" > .circleci-weekly
+      - restore_cache:
+
+          keys:
+            - env-v1-windows-{{ arch }}-py<< parameters.python_version >>-{{ checksum ".circleci/unittest/windows/scripts/environment.yml" }}-{{ checksum ".circleci-weekly" }}
```

Comment:
I think it will be better to add `CUDA_VERSION` to cache key so that CPU unittest environment (which contains `cpuonly`) and CUDA unittest environment (which contains `cudatoolkit`) use different cache.

Reply:
But the cudatoolkit package is not installed in the previous step.

Reply:
oh, you are correct. that's my oversight.